src/Features/Blockcore.Features.BlockStore/Pruning/LeveldbPrunedBlockRepository.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Blockcore.Consensus.BlockInfo;
+using Blockcore.Consensus.Chain;
+using Blockcore.Networks;
+using Blockcore.Utilities;
+using DBreeze.DataTypes;
+using LevelDB;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+
+namespace Blockcore.Features.BlockStore.Pruning
+
+{
+    /// <inheritdoc />
+    public class LeveldbPrunedBlockRepository : IPrunedBlockRepository
+
+    {
+        private readonly IBlockRepository blockRepository;
+        private readonly DataStoreSerializer dataStoreSerializer;
+        private readonly ILogger logger;
+        private static readonly byte[] prunedTipKey = new byte[2];
```

Comment:
What is this field? Why is it 2 bytes? This needs a little comment =)

Reply:
this is a good point (it should probably be 1 byte with the value 3 not similar to the other keys